README.md

```diff
@@ -1,9 +1,9 @@
-# D&D 5e LaTeX Template
+# DnD 5e LaTeX Template
 
 [![Latest release](https://img.shields.io/github/release/evanbergeron/DND-5e-LaTeX-Template/all.svg)](https://github.com/rpgtex/DND-5e-LaTeX-Template/releases/latest)
 [![Build Status](https://travis-ci.org/evanbergeron/DND-5e-LaTeX-Template.svg?branch=master)](https://travis-ci.org/evanbergeron/DND-5e-LaTeX-Template)
 
-This is a LaTeX template for typesetting documents in the style of the *Dungeons & Dragons* 5th Edition (D&D 5e) books.
+This is a LaTeX template for typesetting documents in the style of the fifth edition of the "world's greatest roleplaying game".
```

Comment:
As much as I hate this self-congratulatory turn of phrase, I think we do need to make this change to fully comply with the OGL:

>  "Use", "Used" or "Using" means to use, Distribute, copy, edit, format, modify, translate and otherwise create Derivative Material of Open Game Content

Specifically, the translations we bundle with the package code would fall under "Use". We don't "Use" any other content from the SRD.

It's pretty absurd though, because "Player's Handbook" and "Monster Manual" are also restricted terms. So how do we document that the colours we include are based on those books? 🙄

Reply:
We are not releasing this under the OGL however. The part of the SRD that is released under the OGL (i.e. not "Product Identity") are rules. In this case, I think we should be more worried about trademarks. As far as I can tell, PHB, MM, and DMG are not trademarked, and still work as color names.

Reply:
That being said, I removed the one reference to a book title I could find.